test/integration/project_test.go

```diff
@@ -205,18 +205,20 @@ func TestProjectWatchWithSelectionPredicate(t *testing.T) {
 	}
 	// we should be seeing an "ADD" watch event being emitted, since we are specifically watching this project via a field selector
 	waitForAdd("ns-01", w, t)
+	// TODO: Something is modifying the project, someone should track down what.
+	waitForModified("ns-01", w, t)
```

Comment:
this should no longer be necessary.  Removing it may prevent flakes since we don't know why it's timed this way at the moment.

Reply:
removed, will self-lgtm